playbooks/common/openshift-cluster/additional_config.yml

```diff
@@ -1,3 +1,13 @@
+- name: Configure ntp
+  hosts:
+    - oo_nodes_to_config
+    - oo_etcd_to_config
+  vars:
+    ntp_require_clock_sync: "{{ openshift_require_clock_sync | default(false) | bool }}"
+  roles:
+    - role: ntp
```

Comment:
This is extremely helpful... using wrapper roles and dependencies are the pieces of the puzzle I was failing to realize. Crystal clear now.